### PR TITLE
Add a section for File Managers.

### PR DESCRIPTION
**Pull request by jclopes**
Also removes a duplicate entry for vifm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +122,21 @@ List of projects that provide terminal user interfaces
 ---
 </details>
 
+<details open><summary><h2>File Managers</h2></summary>
+
+- [adbtuifm](https://github.com/darkhz/adbtuifm) A TUI file manager for Android, based on the Android Debug Bridge(ADB).
+- [fff](https://github.com/dylanaraps/fff) A simple file manager written in bash.
+- [fml](https://github.com/wick3dr0se/fml) :file_folder: A stupid simple, fast file manager written in BASH v4.2+.
+- [goful](https://github.com/anmitsu/goful) a powerful TUI file manager written in Go.
+- [lf](https://github.com/gokcehan/lf) A terminal file manager written in Go with heavy inspiration from ranger file manager.
```
Comment (Lockszmith-GH) [inline README.md:131]: Noticed `lf` mentions ranger, but ranger was [left behind](https://github.com/rothgar/awesome-tuis/blob/81159c2cdb02e474efd3f980450d8dd94becc175/README.md?plain=1#L366).

Comment (ccoVeille) [reply README.md:131]: Good catch

File: README.md [hunk 2]
```diff
@@ -288,15 +301,13 @@ List of projects that provide terminal user interfaces
 - [tlock](https://github.com/eklairs/tlock) Two-Factor Authentication Tokens Manager in Terminal
 - [ttyper](https://github.com/max-niederman/ttyper) Terminal-based typing test
 - [tui-shop](https://github.com/Gcat101/tui-shop) Something between a CLI and a GUI way of downloading TUIs/CLIs
-- [vifm](https://github.com/vifm/vifm) A TUI file manager with vi-keybindings and other vim like behaviour.
 - [wb](https://github.com/MertGunduz/wb) A TUI vocabulary notebook app for Linux based devices.
 - [wego](https://github.com/schachmat/wego) Weather app
 - [wavemon](https://github.com/uoaerg/wavemon) A wireless device monitoring application
 - [WG Commander](https://github.com/andrianbdn/wg-cmd) A TUI for a simple WireGuard VPN setup: peer management, QR codes, setup wizard.
 - [wttr.in](https://github.com/chubin/wttr.in) The right way to check the weather
 - [xplr](https://github.com/sayanarijit/xplr) A hackable, minimal, fast TUI file explorer, stealing ideas from nnn and fzf.
```
Comment (ccoVeille) [inline README.md:309]: This one is also a candidate

File: README.md [hunk 3]
```diff
@@ -124,6 +122,21 @@ List of projects that provide terminal user interfaces
...
+- [fml](https://github.com/wick3dr0se/fml) :file_folder: A stupid simple, fast file manager written in BASH v4.2+.
+- [goful](https://github.com/anmitsu/goful) a powerful TUI file manager written in Go.
+- [lf](https://github.com/gokcehan/lf) A terminal file manager written in Go with heavy inspiration from ranger file manager.
+- [nnn](https://github.com/jarun/nnn) n³ The unorthodox terminal file manager.
+- [superfile](https://github.com/MHNightCat/superfile) Pretty fancy and modern terminal file manager.
+- [TUIFIManager](https://github.com/GiorgosXou/TUIFIManager) A cross-platform terminal-based file manager *(supports termux)*.
+- [Vifm](https://github.com/vifm/vifm) A TUI file manager with vi-keybindings and other vim like behaviour.
+- [yazi](https://github.com/sxyazi/yazi) Blazing fast terminal file manager written in Rust, based on async I/O.
+---
+</details>
```
[the remark this answers is not in the document]
Comment (jclopes) [reply README.md:139]: I was expecting to see it here. Don't know why it's not.
Let's keep this change only for the new section and add new entries in separated PRs.

File: README.md (hunk 2 above)
Comment (ccoVeille) [reply README.md:309]: @jclopes @Lockszmith-GH 

What do you think?

Comment (Lockszmith-GH) [reply README.md:309]: yes indeed!

**Pull request merged**